src/test/java/com/wikia/webdriver/testcases/discussions/PollsTests.java

```diff
@@ -0,0 +1,158 @@
+package com.wikia.webdriver.testcases.discussions;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.BasePostsCreator;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.Poll;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.SignInToFollowModalDialog;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.TextGenerator;
+import com.wikia.webdriver.elements.mercury.pages.discussions.PostsListPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+@Test(groups = "discussions-polls")
+public class PollsTests extends NewTestTemplate {
+
+    public static int DEFAULT_ANSWERS_NUMBER = 2;
+
+    @Test
+    @InBrowser(emulator = Emulator.DESKTOP_BREAKPOINT_BIG)
+    @Execute(asUser = User.USER_6, onWikia = MercuryWikis.DISCUSSIONS_2)
+    public void userCanCreatePostWithSimplePollOnDesktop() {
+        PostsListPage page = new PostsListPage().open();
+        BasePostsCreator postsCreator = page.getPostsCreatorDesktop();
+        postsCreator.click().closeGuidelinesMessage().clickAddCategoryButton().selectFirstCategory();
+        postsCreator.addTitleWith(TextGenerator.createUniqueText());
+        Poll poll = postsCreator.addPoll();
+        Assert.assertFalse(postsCreator.isPostButtonActive());
+
+        poll.addTitle(TextGenerator.createUniqueText());
+        Assert.assertFalse(postsCreator.isPostButtonActive());
+        poll.addNthAnswer(TextGenerator.createUniqueText(), 0);
+        Assert.assertFalse(postsCreator.isPostButtonActive()); //poll needs to have at least 2 answers options
```

Comment:
Instead of adding it as a comment, you can set this as assertion message - http://static.javadoc.io/org.testng/testng/6.11/org/testng/Assert.html#assertFalse-boolean-java.lang.String-